dango/testing/tests/dex.rs

```diff
@@ -3725,31 +3825,31 @@ fn volume_tracking_works_with_multiple_orders_from_same_user() {
             user: accounts.user1.username.clone(),
             since: None,
         })
-        .should_succeed_and_equal(Udec128::from_str("300.000146").unwrap());
+        .should_succeed_and_equal(Udec128::from_str("299.999999").unwrap());
```

Comment:
It's better if you leave a comment on where this number comes from. So that if in the future we change contract logics, it's easier to fix this.